src/CommandHelper.ts

```diff
@@ -0,0 +1,89 @@
+import * as os from 'os'
+import * as core from '@actions/core';
+import * as exec from '@actions/exec';
```

Comment:
Reading more about this `@actions/exec` package, I don't think we need to differentiate between PowerShell and Bash based on the OS that the runner executing the action is hosted on. AFAIK, nothing like this was offered for Azure DevOps, so we needed to make this distinction so that we knew which shell to execute commands with, but here the `@actions/exec` library is executing CLI commands in a cross-platform manner, so it doesn't matter what the OS is or what shell is used, the library will figure that out for us.

Reply:
Addressed this comment [here](https://github.com/Azure/container-apps-deploy-action/pull/61#discussion_r1335345938)